test/e2e/specs/editor/various/splitting-merging.spec.js

```diff
@@ -0,0 +1,407 @@
+<!-- /wp:paragraph -->`
+		);
+	} );
+
+	test( 'should ensure always a default block', async ( {
+		editor,
+		page,
+	} ) => {
+		// Feature: To avoid focus loss, removal of all blocks will result in a
+		// default block insertion at the root. Pressing backspace in a new
+		// paragraph should not effectively allow removal. This is counteracted
+		// with pre-save content processing to save post consisting of only the
+		// unmodified default block as an empty string.
+		//
+		// See: https://github.com/WordPress/gutenberg/issues/9626
+		await editor.insertBlock( { name: 'core/paragraph' } );
+		await page.keyboard.press( 'Backspace' );
+
+		// There is a default block and post title:
+		await expect(
+			page.locator( 'role=document[name=/Empty block/i]' )
+		).toBeVisible();
+
+		await expect(
+			page.locator( 'role=textbox[name="Add title"i]' )
+		).toBeVisible();
+
+		// But the effective saved content is still empty:
+		expect( await editor.getEditedPostContent() ).toBe( '' );
+
+		// And focus is retained:
+		await expect(
+			page.locator( 'role=document[name=/Empty block/i]' )
+		).toBeFocused();
+	} );
+
+	test( 'should undo split in one go', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '12' );
+		await page.keyboard.press( 'ArrowLeft' );
+		await page.keyboard.press( 'Enter' );
+		await pageUtils.pressKeyWithModifier( 'primary', 'z' );
+
+		// Check the content.
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'should not split with line break in front', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '1' );
+		await page.keyboard.press( 'Enter' );
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
+		await page.keyboard.type( '2' );
+		await page.keyboard.press( 'ArrowLeft' );
+		await page.keyboard.press( 'Backspace' );
+
+		// Check the content.
+		const content = await editor.getEditedPostContent();
+		expect( content ).toBe(
+			`<!-- wp:paragraph -->
+<p>1</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>2</p>
+<!-- /wp:paragraph -->`
+		);
+	} );
+
+	test.describe(
+		'test restore selection when merge produces more than one block',
+		() => {
+			test( 'on forward delete', async ( {
+				editor,
+				page,
+				pageUtils,
+			} ) => {
+				await editor.insertBlock( { name: 'core/paragraph' } );
+				await page.keyboard.type( 'hi' );
+				await editor.insertBlock( { name: 'core/list' } );
+				await page.keyboard.type( 'item 1' );
+				await page.keyboard.press( 'Enter' );
+				await page.keyboard.type( 'item 2' );
+				await pageUtils.pressKeyTimes( 'ArrowUp', 2 );
+				await page.keyboard.press( 'Delete' );
+				// Carret should be in the first block and at the proper position.
+				await page.keyboard.type( '-' );
+
+				// Check the content.
+				expect( await editor.getEditedPostContent() ).toMatchSnapshot();
```

Comment:
This test used to have two expect checks and now it only has one. Why have these tests been modified?